e2e-tests/cypress/integration/appeal-submission-tasklist.feature

```diff
@@ -1,33 +1,52 @@
-    Given the user "is" the original appellant
-    And the user has provided their name: <name> and email: <email>
-    When the user checks the status of their appeal
-    Then the user should see that the "Your Details" task is <status>
+  Scenario Outline: All the tasks are not started
+    Given an appeal is about to be made
+    When the <task> part of the appeal is not started
+    Then the status of the <task> is "NOT STARTED"
+    Examples:
+      | task                                            |
+      | "About you - Your details"                      |
+      | "Planning application - Application number"     |
+      | "Planning application - Upload application"     |
+      | "Planning application - Upload decision letter" |
+      | "Your appeal - Appeal statement"                |
+      | "Your appeal - Supporting documents"            |
+      | "Appeal site - Site location"                   |
+      | "Appeal site - Site ownership"                  |
+      | "Appeal site - Site access"                     |
+      | "Appeal site - Site safety"                     |
 
+  Scenario Outline: All the tasks are completed
+    Given an appeal has been made
+    When the <task> part of the appeal are completed
+    Then the status of the <task> is "COMPLETED"
     Examples:
-      | name        | email            | status        |
-      | ""          | ""               | "IN PROGRESS" |
-      | ""          | "good@email.com" | "IN PROGRESS" |
-      | "Good Name" | ""               | "IN PROGRESS" |
-      | "Good Name" | "good@email.com" | "COMPLETED"   |
+      | task                                            |
+      | "About you - Your details"                      |
+      | "Planning application - Application number"     |
+      | "Planning application - Upload application"     |
+      | "Planning application - Upload decision letter" |
+      | "Your appeal - Appeal statement"                |
+      | "Your appeal - Supporting documents"            |
+      | "Appeal site - Site location"                   |
+      | "Appeal site - Site ownership"                  |
+      | "Appeal site - Site access"                     |
+      | "Appeal site - Site safety"                     |
 
 
-  Scenario Outline: The status of the Your Details section is accurate
-    Given the user "is not" the original appellant
-    And the user has provided their name: <name> and email: <email>
-    And the user has provided <on behalf of> as the name of the original appellant
-    When the user checks the status of their appeal
-    Then the user should see that the "Your Details" task is <status>
+  Scenario: The status of the "Your details" task is IN PROGRESS
+    Given confirmation of whether appellant is original applicant is requested
+    When confirmation provided that appellant is original applicant
+    Then the status of the "About you - Your details" is "IN PROGRESS"
```

Comment:
Dan asked a question like this and Ian said we should not cover every combination. The purpose is to ensure that these sections could be in progress and the business is never considering the scenario of a wild user to be realistic even if I agree with you...

Reply:
OK. I am moving to the view that this should be a feature file that is concerned only with displaying the correct status badge rather than exercising the rules to create a particular state. I think that should be on other feature files related to specific sections/features. In that case we could use a general scenario outline like Given <section> is <state>, When the appeal tasks are presented, Then the state for <section> is displayed to be <state>. If we take this approach then the various section feature files should be concerned about how we get to be in each of the possible states i.e. the business rules.